internal/core/app_status_report.go

```diff
@@ -345,6 +344,9 @@ func (op *statusReportOperation) Do(
 		return nil, status.Errorf(codes.FailedPrecondition, "unsupported status operation type")
 	}
 
+	// Add the time generated to the outer status report
+	realMsg.GeneratedTime = report.TimeGenerated
```

Comment:
:chefs-kiss:

Reply:
Why is one `AB` and the other `BA` (A = "Generated" B = "Time")?

Renaming fields is backwards compatible in proto so should we rename fields?

Reply:
Good catch...yeah these should be named the same.

Reply:
Ha, I actually [just changed this](https://github.com/hashicorp/waypoint/pull/1594/commits/0132e30ed5828bf2b85f3c46eeafa23238123985) after @jgwhite caught that many similar time fields that the server returns are in the AB order, and it might be better to be consistent with other server fields than with the sdk field that we're mirroring: https://github.com/hashicorp/waypoint/pull/1594#discussion_r645457560

Reply:
If renaming fields is backwards compatible, fixing the SDK field may be the way to go.

Reply:
`realMsg.GeneratedTime` was added in this PR so we should match the naming from the SDK.

https://github.com/hashicorp/waypoint/pull/1594/files#diff-08cc775443acf74b3ff82990db5a6a1dc18774b7701d3851dad9ff91e5947b37R2381

Reply:
Ah, I noticed Jamies comment. We could flip them in the SDK if it doesn't break backwards compatibility. It's too bad this wasn't caught sooner 😄